Rocket.Chat.xcodeproj/project.pbxproj

```diff
@@ -142,7 +142,7 @@
 		412719461E6B403700461FEE /* RegisterUsernameViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 412719451E6B403700461FEE /* RegisterUsernameViewController.swift */; };
 		412731D51DE0A55C00FC45A0 /* ChatPreviewModeView.xib in Resources */ = {isa = PBXBuildFile; fileRef = 412731D41DE0A55C00FC45A0 /* ChatPreviewModeView.xib */; };
 		412731D71DE0A89B00FC45A0 /* ChatPreviewModeView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 412731D61DE0A89B00FC45A0 /* ChatPreviewModeView.swift */; };
-		4129937E208E25C200E12C49 /* PreferencesViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4129937D208E25C100E12C49 /* PreferencesViewController.swift */; };
+		4129937E208E25C200E12C49 /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
👀

Reply:
Also happened in my PR just after adding the missing reference:

https://github.com/RocketChat/Rocket.Chat.iOS/pull/1589

🤷‍♂️